lms/static/scripts/frontend_apps/components/AssignmentToolbar.tsx

```diff
@@ -0,0 +1,54 @@
+import classnames from 'classnames';
+import type { ComponentChildren } from 'preact';
+
+import GradingControls from './GradingControls';
+
+export type AssignmentToolbarProps = {
+  /** Iframe element displaying assignment content. */
+  children: ComponentChildren;
+  courseName: string;
+  assignmentName: string;
+};
+
+/**
+ * Assignment toolbar for instructors. Shows assignment information and grading
+ * controls (for gradeable assignments).
+ */
+export default function AssignmentToolbar({
```

Comment:
Perhaps we should call this `InstructorToolbar`, since it contains controls for use by instructors? This would make the current purpose more obvious. I don't currently foresee a need to make it available to students as well, but we could just rename it again if that happens.